config/resources/test_extensive_config.json

```diff
@@ -0,0 +1,126 @@
+{
+  "aclTokenSecret": {
+    "provider": "secrets-manager",
+    "configuration": {
+      "prefix": "abc123",
+      "consulClientTokenSecretArn": "some-long-arn"
+    }
+  },
+  "mesh": {
+    "bootstrapDir": "/consul/",
+    "healthSyncContainers": ["frontend"],
+    "service": {
+      "name": "frontend",
+      "tags": ["frontend"],
+      "port": 8080,
+      "address": "127.0.0.1",
+      "socketPath": "/path/to/socket",
+      "taggedAddresses": {
+        "lan": {
+          "address": "192.168.1.5",
+          "port": 8080
+        }
+      },
```

Comment:
Not sure these make sense since the addresses can't be static, they need to be dynamic at runtime and also if someone set this to for example the wrong IP then the mesh will be broken.

Reply:
Ah, I removed this for the proxy but not the service. I'm thinking `socketPath` something we should remove here as well?

Reply:
yeah I think so